### ISSUE TITLE
Is there a pytorch version of Pseudo-3d？

### ISSUE THREAD

**Issue by qijiezhao** [ZhaofanQiu/pseudo-3d-residual-networks#4]
sorry to impose,   have you ever reproduced result of P-3d in PyTorch?

**Comment by ZhaofanQiu** [comment]
No. Currently, only the Caffe version is implemented and my next plan is the CNTK platform (since I am during the internship in MSRA).
To run this model on PyTorch, you may need:
  1. Add new layers on PyTorch, and also confirm PyTorch support 5D data (num, channel, length, height, width). (Actually, the bn layer is similar in TSN, and Pooling3D layer is similar in C3D, I think you are already familiar with these works :) )
  2. Convert the Caffe model to PyTorch.
If you want to run on PyTorch, we can have the further discussion since this may be my future plan.

**Comment by qijiezhao** [comment]
Thanks for your early reply, After I have read the details of your proposed paper, I could have a try. Before that, I try to find enough information for C3D training tricks.

**Comment by qijiezhao** [comment]
Hi, Zhaofan:

P-3d seems friendly to implement, I will take time to have a try. If necessary, we can have a detailed chat.

**Comment by ZhaofanQiu** [comment]
Ok, it is my pleasure to help you to do that :)

**Comment by qijiezhao** [comment]
Pytorch version of P3D has been completed. Anyone who needs help can access it as following:
https://github.com/qijiezhao/pseudo-3d-pytorch

**Comment by ahwangyuwei** [comment]
Hi, @qijiezhao .
Could you provide the training code?
Thanks!
